mod-di-converter-storage-server/src/main/java/org/folio/dao/association/ProfileAssociationDao.java

```diff
@@ -99,11 +92,18 @@ Future<Boolean> delete(String masterWrapperId, String detailWrapperId, ProfileTy
    *
    * @param masterId     - UUID of masterProfile
    * @param detailId     - UUID of detailProfile
-   * @param masterType   - master Profile Type
-   * @param detailType   - detail Profile Type
+   * @param masterType a master type in association
+   * @param detailType a detail type in association
    * @param tenantId     - tenant id
    * @return - boolean result of operation
    */
   Future<Boolean> deleteByMasterIdAndDetailId(String masterId, String detailId, ProfileType masterType,
                                               ProfileType detailType, String tenantId);
+
+  /**
+   * Check if profile_association table is empty
+   * @param tenantId - tenant id
+   * @return future with if delete = true, if not - false
+   */
+  Future<Boolean> checkIfDataInTableExists(String tenantId);
```

Comment:
could you please say do we call this method somewhere?

Reply:
I decided to remove it later, because, with this check association migration is not running.